src/instance/mod.rs

```diff
@@ -18,27 +18,39 @@ use data::{Map,Monster};
 
 pub mod management;
 
+// Computation of the duration of one so-called 'tick'. A 'tick' is a
+// unite in the discrete time of lycan. By default, its value is set
+// to about 16ms.
```

Comment:
I looked at what other online game engines are doing, and I have seen 60Hz, 30Hz (for FPS based on the Source engine), and down to 1Hz (for Eve Online).

I agree that it should be based on experimentation, but as long as we don't see any problems I would suggest leaving it as is.

I think right now no more variables are making any assumption about this value, so we should be able to change it at any time.